management_api_app/api/routes/workspace_templates.py

```diff
@@ -3,32 +3,81 @@
 from fastapi import APIRouter, Depends, HTTPException
 from starlette import status
 
+from models.domain.resource_template import ResourceTemplate
+from models.schemas.template import TemplateInCreate, TemplateInResponse
+from models.schemas.workspace_service_template import WorkspaceServiceTemplateInCreate, \
+    WorkspaceServiceTemplateInResponse
 from services.authentication import get_current_admin_user
 from api.dependencies.database import get_repository
 from db.errors import EntityDoesNotExist
 from db.repositories.workspace_templates import WorkspaceTemplateRepository
-from models.schemas.workspace_template import (WorkspaceTemplateNamesInList, WorkspaceTemplateInCreate, WorkspaceTemplateInResponse)
+from models.schemas.workspace_template import (WorkspaceTemplateNamesInList, WorkspaceTemplateInCreate,
+                                               WorkspaceTemplateInResponse)
+from models.domain.resource import ResourceType
 from resources import strings
-from services.concatjsonschema import enrich_schema_defs
+from services.concatjsonschema import enrich_schema_defs, enrich_workspace_service_schema_defs
```

Comment:
nit: for consistency, should we rename enrich_schema_defs to enrich_workspace_schema_defs at the same time